### PR TITLE
[RELEASE_8.14.0] Add new field `generateDocumentOutline`

### PR DESCRIPTION
**Pull request by StevenRenaux**

### CODE REVIEW

**Review by Neirda24 — APPROVED** [review]

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -229,6 +229,10 @@ private function addChromiumPdfOptionsNode(ArrayNodeDefinition $parent): void
                     ->info('Define whether to prefer page size as defined by CSS - default false. https://gotenberg.dev/docs/routes#page-properties-chromium')
                     ->defaultNull()
                 ->end()
+                ->booleanNode('generate_document_outline')
+                    ->info('Define whether the document outline should be embedded into the PDF - default false. https://gotenberg.dev/docs/routes#page-properties-chromium')
+                    ->defaultNull()
```
Comment (Jean-Beru) [inline src/DependencyInjection/Configuration.php:234]: Is it working with the default value since `generateDocumentOutline` does not accept `null` value ?

Comment (Neirda24) [reply src/DependencyInjection/Configuration.php:234]: null and [] are filtered out in the Extension file. So it is not set to null. It is simply ignored by the configuration.

Comment (StevenRenaux) [reply src/DependencyInjection/Configuration.php:234]: https://github.com/sensiolabs/GotenbergBundle/blob/bbdfaebcc6396a3a5ce1cd2c1ea11f497c63fdd1/src/DependencyInjection/SensiolabsGotenbergExtension.php#L132

Comment (Jean-Beru) [reply src/DependencyInjection/Configuration.php:234]: :ok_hand:

**Review by Jean-Beru — APPROVED** [review]

**Pull request merged**